config/locales/en/admin.yml

```diff
@@ -1648,6 +1648,7 @@ en:
           create_card: Create card
           no_cards: There are no cards.
           columns_help: "Width of the card in number of columns. On mobile screens it's always a width of 100%."
+          order_help: "You can enter the position where this card will be shown to the user in the homepage"
```

Comment:
I think we can remove "to the user" :thinking:.

On the other hand, I wonder whether it makes sense to mention "homepage" when sometimes we mean a custom page and sometimes we mean the SDG index :thinking:.